site/data/sidebar.yml

```diff
@@ -52,11 +52,11 @@
     - title: Navs
     - title: Navbar
     # Boosted mod
```

Comment:
The elements in this `# Boosted mod`/`# end mod` don't respect the alphabetical order that seems to be in place here in 'Components'. Would it be possible to use this PR to also reorder those 5 components in the list at the right place?